.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,48 @@
+# Workflow to validate commits to an open PR.
+
+name: Validate PR
+
+env:
+  INSTALL_EDM_VERSION: 3.2.3
+  PYTHON_VERSION: 3.6
+  PYTHONUNBUFFERED: 1
+  QT_DEBUG_PLUGINS: 1
+
+on:
+  pull_request
+
+jobs:
+  test-with-edm:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        toolkit: ['pyqt', 'pyqt5', 'pyside2']
```

Comment:
We've stopped testing against pyqt in other ETS projects and we can probably stop here too. I'll defer the final decision to @stevenjkern .